feature-utils/poly-look/src/css/styles.css

```diff
@@ -44,3 +44,9 @@ so that we don't have to cover three different use cases explicitly anymore.
 .poly-nav-bar-separator-bottom {
   box-shadow: 0px 1px #a9b6c6;
 }
+
+/* Size */
+:root {
+  --max-width: 412px;
+  --line-height: 120%;
+}
```

Comment:
Rather put this in themes.css (line-height is already there)

Reply:
I guess the max-width will at some point go into a theme like "mobile or something"

Reply:
Done. Also 100% agree that we'll have to move it at some point. How/where I don't know yet.